tests/ConfigR.Tests.Acceptance.Roslyn.CSharp/Support/ConfigFile.cs

```diff
@@ -27,7 +27,7 @@ public static IDisposable Create(string contents, string path)
         public static string GetDefaultPath()
         {
             AppDomain.CurrentDomain.SetData("APP_CONFIG_FILE", "Test.exe.config");
-            return Path.ChangeExtension(AppDomain.CurrentDomain.SetupInformation.ConfigurationFile, "csx");
+            return AppDomain.CurrentDomain.SetupInformation.ConfigurationFile;

```

Comment:
This was a mistake. The app domain configuration file should never have been set to a csx file. This was causing https://github.com/config-r/config-r/blob/c66205b8d86881d49108e64e99b45df1f611902a/tests/ConfigR.Tests.Acceptance.Roslyn.CSharp/SearchPathsFeature.cs#L67 to fail. The current version of `HttpClient` was trying to read from the csx file and it was throwing an exception saying that it is malformed. Earlier versions of `HttpClient` must have been more forgiving.